### ISSUE TITLE
should normalize `url` and `context`

### ISSUE THREAD

**Issue by TerenceZ** [jtangelder/sass-loader#94]
In windows~~, this loader will make the webpack stuck if involving paths like '../../some path'.~~  __this loader will let the webpack resolve wrong paths if involving paths like '../../some path'.__ To work around this, I find that just normalize the `url` and `context` in proper position:

```js
function asyncResolve(loaderContext, url, context, done) {
    context = path.normalize(context);
    if (url.charAt(0) === '.' && url.charAt(1) !== '.') {
        url = '.' + path.normalize(url.slice(1));
    } else {
        url = path.normalize(url);
    }
    loaderContext.resolve(context, url, function onWebpackResolve(err, filename) {
        // ...
    });
}
```

The test version is 0.6 and 1.0.1.

**Comment by TerenceZ** [comment]
Sorry, I find this is not the reason to cause the problem...

**Comment by TerenceZ** [comment]
Although this won't make webpack stuck, but it will still let webpack resolve wrong path.

**Comment by jhnns** [comment]
That's strange. The sass-loader doesn't modify any paths. Webpack should also be possible to work with these paths. Could you find out how these paths are messed up? What is actually wrong?

**Comment by jorrit** [comment]
Is this still an issue? Can you create a PR with just a test that fails because of this bug?

**Comment by jhnns** [comment]
Should be fixed with 2.0.0
